pod/playlist/views.py

```diff
@@ -76,6 +77,72 @@ def playlist(request, slug=None):
         )
 
 
+# @login_required
+# @csrf_protect
+def playlist_play(request, slug=None):
+    template_video = (
+        "playlist_player-iframe.html"
+        if (request.GET.get("is_iframe"))
+        else "playlist_player.html"
+    )
+    playlist = get_object_or_404(Playlist, slug=slug) if slug is not None else None
+    if playlist and request.user != playlist.owner and not playlist.visible:
+        # not (request.user.is_superuser or request.user.has_perm(
+        #        "video.change_theme")
+        messages.add_message(
+            request,
+            messages.ERROR,
+            _("You don't have access to this playlist."),
+        )
+        raise PermissionDenied
+    video = None
+    position = (
+        int(request.GET.get("p", 1)) - 1
+        if int(request.GET.get("p", 1)) > 0
+        and int(request.GET.get("p", 1)) <= playlist.playlistelement_set.all().count()
+        else 0
+    )
```

Comment:
peut etre prévoir `try except` ou `isnumeric` ici dans le cas où le `request.GET.get("p", 1)` ne peut être converti en `int`

Reply:
OK j'ajoute un test isnumeric()
int(request.GET.get("p", 1)) - 1
        if request.GET.get("p", 1).isnumeric()
        and int(request.GET.get("p", 1)) > 0
        and int(request.GET.get("p", 1)) <= playlist.playlistelement_set.all().count()
        else 0